services/motion/client.go

```diff
@@ -140,37 +140,63 @@ func (c *client) MoveOnGlobe(
 		req.Obstacles = obstaclesProto
 	}
 
-	if !math.IsNaN(motionCfg.LinearMPerSec) && motionCfg.LinearMPerSec != 0 {
-		req.MotionConfiguration.LinearMPerSec = &motionCfg.LinearMPerSec
+	resp, err := c.client.MoveOnGlobe(ctx, req)
+	if err != nil {
+		return false, err
 	}
-	if !math.IsNaN(motionCfg.AngularDegsPerSec) && motionCfg.AngularDegsPerSec != 0 {
-		req.MotionConfiguration.AngularDegsPerSec = &motionCfg.AngularDegsPerSec
+
+	return resp.Success, nil
+}
+
+func (c *client) MoveOnGlobeNew(
+	ctx context.Context,
+	componentName resource.Name,
+	destination *geo.Point,
+	heading float64,
+	movementSensorName resource.Name,
+	obstacles []*spatialmath.GeoObstacle,
+	motionCfg *MotionConfiguration,
+	extra map[string]interface{},
+
+	// note from Ray it feels weird that we're returning a string here when its an "id"
```

Comment:
Want to start a comment thread for this.  I missed in the original scope that this was going to be a string type rather than an int and I am wondering why we decided this.